tensorrt_llm/_torch/distributed/ops.py

```
@@ -531,7 +580,7 @@ class MNNVLAllReduce(nn.Module):
     fast path that keeps the local value in registers and volatile-loads only peers
     from the Lamport buffer. Larger world sizes use a compact deterministic fallback.
     """
-    allreduce_mnnvl_workspaces: Dict[Mapping, Dict] = {}
+    allreduce_mnnvl_workspaces: dict[Mapping, _MnnvlWorkspace] = {}
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)